packages/docs/stories/components/select.stories.ts

```diff
@@ -0,0 +1,404 @@
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const HelpText: Story = {
+  name: 'Help Text',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'help-text'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select label="Experience" help-text="Please tell us your skill level.">
+      <syn-option value="1">Novice</syn-option>
+      <syn-option value="2">Intermediate</syn-option>
+      <syn-option value="3">Advanced</syn-option>
+    </syn-select>
+  `,
+};
+
+export const Placeholders: Story = {
+  name: 'Placeholders',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'placeholder'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select placeholder="Select one">
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const Clearable: Story = {
+  name: 'Clearable',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'clearable'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select clearable value="option-1">
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const Disabled: Story = {
+  name: 'Disabled',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'disabled'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select placeholder="Disabled" disabled>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+    </syn-select>
+  `,
+};
+
+export const Multiple: Story = {
+  name: 'Multiple',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'multiple'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select label="Select a Few" value="option-1 option-2 option-3" multiple clearable>
+      <syn-option value="option-1">Option 1</syn-option>
+      <syn-option value="option-2">Option 2</syn-option>
+      <syn-option value="option-3">Option 3</syn-option>
+      <syn-option value="option-4">Option 4</syn-option>
+      <syn-option value="option-5">Option 5</syn-option>
+      <syn-option value="option-6">Option 6</syn-option>
+    </syn-select>
+  `,
+};
+
+export const SettingInitialValues: Story = {
+  name: 'Setting Initial Values',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'initialvalue'),
```

Comment:
There seems to be a typo in our doc token:. The `s` is missing:

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/a009147b-7e68-432a-b968-70384caa5d1c)